package-lock.json

```diff
@@ -446,12 +446,56 @@
           "resolved": "https://registry.npmjs.org/babel-register/-/babel-register-6.26.0.tgz",
           "integrity": "sha1-btAhFz4vy0htestFxgCahW9kcHE=",
           "requires": {
+            "babel-core": "6.26.0",
```

Comment:
don't modify the package.lock file

Reply:
Yeah, I thought I removed that sorry. It's pretty confusing (and IMHO wrong unless I'm misunderstanding how it should work) that `npm` will generate different package lock files on different machines without having modified any of the packages in my PR. I've found that Yarn is much more stable about its lock file, and it doesn't magically change between machines.